Updated the SIWA QuickStart [SDK-1489] (#8872)
auth0/docs: articles/quickstart/native/ios-swift-siwa/_includes/_login_centralized.md

}
```

> The `fullName` parameter is needed for Auth0 to be able to complete the user profile with properties like the first and last name, because the ID Token obtained from Apple does not include that information.

Reviewer: Let's use the active voice here and make it more concise:

```suggestion
Auth0 requires the `fullName` parameter to complete the user profile with properties like the first and last name, because the ID Token obtained from Apple does not include that information.
```
